src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -55,6 +55,7 @@ public AddPersonCommand parse(String args) throws ParseException {
             priceRange = ParserUtil.parsePriceRange(argMultimap.getValue(PREFIX_PRICE_RANGE).get());
         }
 
+        // TODO: Consider allowing multiple -c instead of separated by ; in one -c
```

Comment:
Maybe we should also add a TODO for `AddPropertyCommandParser` for the same issue?

Opened an issue #114 for this as well!